app-misc/fhem/fhem-5.8.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user
+
+DESCRIPTION="A GPL'd perl server for house automation"
+HOMEPAGE="https://www.fhem.de/"
+SRC_URI="https://www.fhem.de/${P}.tar.gz"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="doc"
+
+RDEPEND="dev-perl/Crypt-CBC
+	dev-perl/Device-SerialPort
+	dev-perl/Digest-CRC
+	dev-perl/JSON"
+
+DEPEND="media-gfx/pngcrush"
+
+S="${WORKDIR}/${P}"
+
+QA_PREBUILT="opt/fhem/contrib/lcd4linux/fritzbox_dpf/lcd4linux
+	opt/fhem/contrib/lcd4linux/rpi_dpf/lcd4linux"
+
+pkg_setup() {
+	# Create FHEM user and group
+	enewgroup fhem
+	enewuser fhem -1 -1 /opt/fhem fhem
+}
+
+src_prepare() {
+	# Fix install path in Makefile
+	sed -i -e 's,^\(BINDIR=\).*,\1'${D}/opt/fhem',' Makefile || die
+
+	# Remove docs in Makefile
+	sed -i -e 's/docs//g' Makefile || die
+	sed -i -e '/README_DEMO.txt/d' Makefile || die
+
+	# Remove manpage in Makefile
+	sed -i -e '/fhem.pl.1/d' Makefile || die
+
+	# Remove log dir, as it will be replaced with a symlink
+	rm -r log || die
+
+	# Copy config
+	cp "${FILESDIR}"/fhem.cfg fhem.cfg || die
+
+	# Fix fhemicon_darksmall.png, as it reports "broken IDAT window length"
+	pngcrush -fix -force -ow www/images/default/fhemicon_darksmall.png || die
+
+	# Apply user patches
```

Comment:
useless comment

Reply:
Well, that's open for discussion ;-) I don't think, that this will bother anyone. Anyway.. removed this (and the other) comments.